collections/Arch_Selden/MS_Arch_Selden_A_2.xml

```diff
@@ -30,22 +32,37 @@
                   <institution>University of Oxford</institution>
                   <repository>Bodleian Library</repository>
                   <idno type="shelfmark">MS. Arch. Selden. A. 2</idno>
-                  <idno type="ieArk">ark:29072/x0d504rk54z7</idno><idno type="crArk">ark:29072/x0t722h840j8</idno><altIdentifier type="internal">
+                  <idno type="ieArk">ark:29072/x0d504rk54z7</idno>
+                  <idno type="crArk">ark:29072/x0t722h840j8</idno>
+                  <altIdentifier type="internal">
                      <idno type="SCN">3135</idno>
                   </altIdentifier>
+                  <msName>Codex Selden</msName>
+                  <msName xml:lang="es">Códice Selden</msName>
+                  <msName>Codex Añute</msName>
+                  <msName xml:lang="es">Códice Añute</msName>
                </msIdentifier>
+               <head>Codex Selden or Codex Añute: Mixtec pictorial manuscript (<foreign xml:lang="xtd">ñee ñuhu</foreign>)</head>
                <msContents>
                   <msItem class="#historica" n="1" xml:id="MS_Arch_Selden_A_2-item1">
-                     <title key="work_11055" type="desc">'Codex Selden': Mexican pictorial screenfold manuscript</title>
-                     <note>Of Mixtec origin, pre-Conquest in style, recording the genealogies and history from A.D. 783 to A.D. 1556 of the rulers of a place called 'Cloud Belching Mountain', identifiable as Añute (<placeName key="place_7380762">Magdalena Jaltepec</placeName>) in the Mixtec Alta (Oaxaca, Mexico).</note>
-                     <!-- ?no language, just pictorial -->
-                     <textLang mainLang="zxx">No linguistic content</textLang>
+                     <title key="work_11055" type="desc">History of the dynasty that ruled Añute</title>
+                     <note>Genealogies and history of the rulers of Añute (<placeName key="place_7380762">Magdalena Jaltepec</placeName>), in the Valley of Atoco (Nochixtlan), in Ñuu Dzaui Ñuhu, the Mixteca Alta. The last ruler portrayed is Iya Sicuañe (Lord 10 Grass), concluding in the year 11 Flint (which corresponds to 1556).</note>
+                     <textLang mainLang="xtd">Mixtec pictograms</textLang>
                   </msItem>
                </msContents>
+               
                <physDesc>
-                  <objectDesc form="codex">
+                  <objectDesc form="screenfold">
                      <supportDesc material="perg">
                         <support>deerskin</support>
+                        <extent>
+                           <measure>20 pages</measure>
```

Comment:
is "pages" ok? I used it with scare quotes for MS. Laud MIsc. 678 (q.v.), but would "leaves" in scare quotes be better, as in each case we are talking about objects with "recto" and "verso" sides